### ISSUE TITLE
Download failed: `[object Object]`

### ISSUE THREAD

**Issue by XAMPPRocky** [koenvanzuijlen/giantbomb-show-dl#30]
Hello again, I've been using the new 2.2.0 version, and thanks to the new functionality I noticed there some pages it's been trying to download that a way further back, and I've noticed a couple of them fail with the error message `[object Object]`. These downloads fail immediately (like within moments of trying to download them.) and consistently between archive attempts.

Let me know if there's anything I can do to help debug this further.

```
Downloading Jam Along With the Myriad Fighters of Tekken Tag Tournament 2
	Downloading video to: 2012-04-17 12_00_00 - Jam Along With the Myriad Fighters of Tekken Tag Tournament 2.mp4
	Error: Download failed: [object Object]
```

```
Downloading E3 2012: Signal Studios Introduces Ascend: New Gods
	Downloading video to: 2012-06-05 18_44_00 - E3 2012_ Signal Studios Introduces Ascend_ New Gods.mp4
	Error: Download failed: [object Object]
```

```
Downloading In Sonic & All-Stars Racing Transformed, You'll Race Over Land, Seas, and Sky
	Downloading video to: 2012-07-12 11_49_00 - In Sonic & All-Stars Racing Transformed, You'll Race Over Land, Seas, and Sky.mp4
	Error: Download failed: [object Object]
```

**Comment by koenvanzuijlen** [comment]
These videos are also broken on the website

https://www.giantbomb.com/videos/jam-along-with-the-myriad-fighters-of-tekken-tag-t/2300-5883/
https://www.giantbomb.com/videos/e3-2012-signal-studios-introduces-ascend-new-gods/2300-6154/
https://www.giantbomb.com/videos/in-sonic-all-stars-racing-transformed-youll-race-o/2300-6329/

That is probably the cause, I can look into making the error message better. 

But skipping these videos for future runs might be problematic if they ever get fixed. This will probably not happen for 2012 videos, but it might be an issue for more recent cases.

**Comment by XAMPPRocky** [comment]
> That is probably the cause, I can look into making the error message better.
> 
> But skipping these videos for future runs might be problematic if they ever get fixed. This will probably not happen for 2012 videos, but it might be an issue for more recent cases.

Thank you for your response, I agree that the solution here is a better error message, so it's clear what's happening.

**Comment by koenvanzuijlen** [comment]
Fixed in version 2.2.1. Note that the error just shows a "Request failed 403", that's all the info I get on my side.
